docs/support-tasks.md

```diff
@@ -29,6 +29,19 @@ $ bundle exec rake manage:change_email_address[<old_email_address>, <new_email_a
 
 [change]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:change_email_address[from@example.org,to@example.org]
 
+## View subscriber's recent emails
+
+This task shows the most recent email delivery attempts made to the given user.
+The history limit is 10 emails by default, but you can override this.
+
+```bash
+$ bundle exec rake manage:view_emails[<email_address>,<limit=10>]
+```
+
+[⚙ Run rake task on production][view_emails]
+
+[view_emails]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:view_emails[email@example.org,10]
```

Comment:
Related - do you think we should omit the `,10` from the URL and rely on devs to add a second parameter if needed?

```suggestion
[view_emails]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:view_emails[email@example.org]
```

Reply:
Yeah. I think in most cases the default of 10 will suffice.

Reply:
It's nice to have the option of changing it though.